README.md

```diff
@@ -30,7 +30,22 @@ Pravega Presto connector has a set of unit tests that can take several minutes t
 
     ./gradlew test
 
-## Running Presto
+## Installing Presto
+
+Instructions for downloading, installing and configuring Presto can be found here: https://prestodb.io/docs/current/installation/deployment.html.
+
+A typical Presto installation installs the Presto server in /usr/lib. You download (via wget) the gzip'd tar file from Maven using the link defined in the PrestoDB deployment section.  Untar the file while in /usr/lib, which will create a presto-server-<RELEASE> directory.  Create a symbolic link to the code as follows
+    
+    # [root@lrmk226 ~]# ln -s /usr/lib/presto-server-<RELEASE> /usr/lib/presto
```

Comment:
I didn't have to do this.

Reply:
OK, but these instructions are setting it up so that you can drop a new Presto gzip tar distribution in and just recreate some symlinks, otherwise you have to A) change your path to have the launcher in your PATH and B) copy config files from etc in old package to the newly installed package.